dev/bin/manager-run.sh

```diff
@@ -26,6 +26,10 @@ export KAFKA_CLIENT_SECRET=${ops_client_secret}
 export MANAGED_CONNECTORS_KAFKA_CLIENT_ID=${mc_client_id}
 export MANAGED_CONNECTORS_KAFKA_CLIENT_SECRET=${mc_client_secret}
 
+rm -rf ${LOCAL_ENV_FILE}
+echo "MANAGER_URL=http://localhost:8080" >> ${LOCAL_ENV_FILE}
```

Comment:
is that url correct? in QE internal discussion we mentioned it should be `minikube ip`/manager